src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -58,6 +58,9 @@ if [ "$BASE_URL" == "" ]; then
     # Set SSL verify to False
     ./bin/ceph dashboard set-rgw-api-ssl-verify False
 
+    # Disable PG autoscaling for pool PG editing
```

Comment:
Can you add a note that this is a temporary workaround and we would need to change it again if https://tracker.ceph.com/issues/38227 is addressed?

Reply:
Thanks. Fixed.